.circleci/config.yml

```diff
@@ -59,7 +59,7 @@ jobs:
           name: Unit Tests
           command: |
             source python/bin/activate
-            make -C package unit-tests --makefile ../Makefile
```

Comment:
The featuretools CI doesn't use `make test` so that the unit tests will run evein if the lint tests fail.  We should do the same here